kovalenko/home-work-01/src/MathUtils.java

```diff
@@ -0,0 +1,12 @@
+import java.util.Arrays;
+
+public class MathUtils {
+
+    public static int min(int[] array) {
+        return Arrays.stream(array).min().getAsInt();
```

Comment:
I'd prefer manual implementation of minimum, not using stream api, but still okay.